From 8d3eacd7b0bfe36bd68b03895235fe5ba01c6ce9 Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Wed, 26 Dec 2018 13:44:21 +0100
Subject: [PATCH 7/8] selinux: move vendor_camera_prop to
 device/lineage/sepolicy

Snap wants to read vendor specific camera props which are not just a qcom thing.
To be able to grant Snap the right to read those props without breaking non-qcom devices
the definition for vendor_camera_prop should be moved to global lineage ruleset.

Change-Id: I3316613c3f950a9989da2de5ffc87db7efbcb7aa
---
 vendor/common/property.te       | 3 ++-
 vendor/common/property_contexts | 5 +++--
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/vendor/common/property.te b/vendor/common/property.te
index 236a90f9..8328593f 100644
--- a/vendor/common/property.te
+++ b/vendor/common/property.te
@@ -182,7 +182,8 @@ type mm_parser_prop, property_type;
 type mm_video_prop, property_type;
 
 #vendor-camera
-type vendor_camera_prop, property_type;
+#moved to device/lineage/sepolicy
+#type vendor_camera_prop, property_type;
 
 #rmt_storage
 type ctl_vendor_rmt_storage_prop, property_type;
diff --git a/vendor/common/property_contexts b/vendor/common/property_contexts
index 81472b09..666ff6d7 100644
--- a/vendor/common/property_contexts
+++ b/vendor/common/property_contexts
@@ -228,8 +228,9 @@ vendor.hevc.mux.print.codec.cfg    u:object_r:mm_video_prop:s0
 ro.vendor.mm-enable-accurate-seek  u:object_r:mm_video_prop:s0
 
 #vendor-camera
-vendor.camera.aux.packagelist      u:object_r:vendor_camera_prop:s0
-persist.vendor.camera.privapp.list u:object_r:vendor_camera_prop:s0
+#moved to device/lineage/sepolicy
+#vendor.camera.aux.packagelist      u:object_r:vendor_camera_prop:s0
+#persist.vendor.camera.privapp.list u:object_r:vendor_camera_prop:s0
 
 #rmt_storage
 ctl.vendor.rmt_storage     u:object_r:ctl_vendor_rmt_storage_prop:s0
-- 
2.17.1

